flickr_download/flick_download.py

```diff
@@ -317,6 +317,25 @@ def serialize_json(obj):
     return ret
 
 
+def get_photo_sizes(photo):
+    pSize = {}
+    for attempt in range(1, 5):
+        try:
+            pSize = photo.getSizes()
```

Comment:
you can change this to `return photo.getSizes()` and then delete the initialization of `pSize` above.